app/event/send-session-event.js

```diff
@@ -13,7 +13,7 @@ const sendSessionEvent = async (organisation, sessionId, entryKey, key, value, i
       data: { [key]: value },
       ip
     }
-    await raiseEvent(event)
+    raiseEvent(event)
```

Comment:
This wont need to be an async function and as such the call to it can also have the await removed.